test/helm-test/main.go

```diff
@@ -232,6 +232,14 @@ func doMain() int {
 			return execErr
 		})
 
+		xmlWrap(fmt.Sprintf("Helm Update Deps %s", path.Base(chartPath)), func() error {
+			o, execErr := output(exec.Command(helmPath, "dep", "update", chartPath))
```

Comment:
we probably should do a `dep build` instead of `update` as `build` will use the lockfile instead of updating the dependency